R/package.R

```diff
@@ -95,7 +96,8 @@ initialize_python <- function(required_module = NULL, use_environment = NULL) {
                 config$virtualenv_activate,
                 config$version >= "3.0",
                 interactive(),
-                numpy_load_error)
+                numpy_load_error,
+                as.numeric(trace_ms))
```

Comment:
This should probably be `as.integer` since that the type you end up with in C++.

Reply:
changed.